sql/privilege.md

```diff
@@ -0,0 +1,311 @@
+---
+Title: privilege management
+Category: compatibility
+---
+ 
+ 
+# Privilege Management
+ 
+TiDB's privilege management system is implemented according to the privilege management system in MySQL. It supports most of the syntaxes and privilege types in MySQL. If you find any inconsistency with MySQL, feel free to [open an issue](https://github.com/pingcap/docs-cn/issues/new).
+ 
+**Note:** Starting from RC3, the privilege management function is enabled by default. But if you use earlier versions than RC3, you must add the following startup option or the privilege check does not work: 
+
+	./tidb-server -privilege=true
+
+## 1. User account operation
+
+TiDB user account names consist of a user name and a host name. The account name syntax is `'user_name'@'host_name'`.
+
+The `user_name` is case sensitive. 
+The `host_name` can be a host name or an IP address. The `%` and `_` wildcard characters are permitted in host name or IP address values. For example, a host value of `'%'` matches any host name and `'192.168.1.%'` matches every host on a subnet. 
+
+### Create user
+
+The `CREATE USER` statement creates new MySQL accounts. 
+ 
+```sql
+create user 'test'@'127.0.0.1' identified by 'xxx';
+``` 
+
+If the host name is not specified, you can log in from any IP address. If the password is not specified, it is empty by default:
+ 
+```sql
+create user 'test';
+```
+ 
+Equals:
+ 
+```sql
+create user 'test'@'%' identified by '';
+```
+
+**Required Privilege:** To use `CREATE USER`, you must have the global `CREATE USER` privilege.
+
+### Change the password
+
+You can use the `SET PASSWORD` syntax to assign or modify a password to a user account.
+ 
+```sql
+set password for 'root'@'%' = 'xxx';
+```
+
+**Required Privilege:** Operations that assign or modify passwords are permitted only to users with the `CREATE USER` privilege.
+ 
+### Drop user
+
+The `DROP USER` statement removes one or more MySQL accounts and their privileges. It removes the user record entries in the `mysql.user` table and the privilege rows for the account from all grant tables.
+ 
+```sql
+drop user 'test'@'%';
+```
+**Required Privilege:** To use `DROP USER`, you must have the global `CREATE USER` privilege.
+ 
+### Reset the root password
+
+If you forget the root password, you can skip the privilege system and use the root privilege to reset the password.
+
+To reset the root password,
+
+1. Start TiDB with a special startup option (root privilege required)：
+ 
+	```bash
+	sudo ./tidb-server -skip-grant-table=true
+	```
+ 
+2. Use the root account to log in and reset the password:
+ 
+	```base
+	mysql -h 127.0.0.1 -P 4000 -u root
+	```
+ 
+## 2. Privilege-related operations
+ 
+### Grant privileges
+
+The `GRANT` statement grants privileges to the user accounts. 
+
+For example, use the following statement to grant the `xxx` user the privilege to read the `test` database.
```

Comment:
Actually, we need to surround the name with "the" and the user. Same is true to command names like the xx command, etc.